ding/framework/context.py

```diff
@@ -49,8 +50,12 @@ class OnlineRLContext(Context):
     env_episode: int = 0
     train_iter: int = 0
     train_data: Union[Dict, List] = None
+    train_output: Union[Dict, List] = None
```

Comment:
Dict -> List[Dict]

Reply:
train_output has two cases, in trainer it is Dict, in learner it is List[Dict]. Should we only define the case of learner here?

Reply:
Union[Dict, List[Dict]]